packages/components-js/tests/e2e/puppeteer/specs/modal.e2e.ts

```diff
@@ -466,6 +468,24 @@ describe('focus behavior', () => {
       await page.keyboard.press('Tab');
       await expectDismissButtonToBeFocused('finally');
     });
+
+    it('should not set scrollTop = 0 on changes within modal children', async () => {
+      await goto(page, 'modal-change-content');
```

Comment:
I think during e2e tests the app is neither built, nor served.
Also the example pages bring no benefit to reproduce the bahvior.